WordPress/src/main/java/org/wordpress/android/ui/domains/management/DomainsListCard.kt

```diff
@@ -0,0 +1,174 @@
+import androidx.compose.material3.LocalTextStyle
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.material3.OutlinedCard
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment.Companion.CenterVertically
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import org.wordpress.android.R
+import org.wordpress.android.fluxc.network.rest.wpcom.site.AllDomainsDomain
+import org.wordpress.android.fluxc.network.rest.wpcom.site.DomainStatus
+import org.wordpress.android.fluxc.network.rest.wpcom.site.StatusType
+import java.time.LocalDate
+import java.time.ZoneId
+import java.util.Date
+
+@Composable
+fun DomainListCard(uiState: DomainCardUiState) {
+    OutlinedCard(
+        modifier = Modifier.fillMaxWidth(),
+        border = BorderStroke(1.dp, MaterialTheme.colorScheme.outlineVariant),
+    ) {
+        Row(
+            verticalAlignment = CenterVertically,
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(16.dp),
+        ) {
+            Column(
+                modifier = Modifier.weight(1f),
+            ) {
+                when (uiState) {
+                    DomainCardUiState.Initial -> {
+                        PendingGhostStrip(width = 100.dp)
+                        Spacer(modifier = Modifier.height(4.dp))
+                        PendingGhostStrip(100.dp)
+                        Spacer(modifier = Modifier.height(8.dp))
+                        StatusRow(
+                            uiState = StatusRowUiState.Initial,
+                            modifier = Modifier.fillMaxWidth(),
+                        )
+                    }
+                    is DomainCardUiState.Loaded -> {
+                        uiState.domain?.let { domain ->
+                            Text(
+                                text = domain,
+                                style = MaterialTheme.typography.bodyLarge,
+                            )
+                        }
+                        Spacer(modifier = Modifier.height(4.dp))
+                        uiState.title?.let {
+                            Text(
+                                text = it,
+                                color = MaterialTheme.colorScheme.outline,
+                            )
+                        }
+                        Spacer(modifier = Modifier.height(8.dp))
+                        StatusRow(
+                            uiState = uiState.statusUiState,
+                            modifier = Modifier.fillMaxWidth(),
+                        )
+                    }
+                }
+            }
+            Spacer(modifier = Modifier.width(16.dp))
+            Icon(
```

Comment:
This is a suggestion, not a change request, and can be handled on a follow up PR. Would it make sense to hide the Icon while loading in the Initial ui state?

Reply:
Good point. I'll defer to Paul, but we can address in one of the followup PRs if needed.